pkg/stub/handler.go

```diff
@@ -240,3 +244,58 @@ func (h *Handler) ensureDNSDeleted(dns *dnsv1alpha1.ClusterDNS) error {
 	}
 	return nil
 }
+
+// syncClusterDNSStatus updates the status for a given clusterdns.
+func syncClusterDNSStatus(dns *dnsv1alpha1.ClusterDNS, clusterIP, clusterDomain string) error {
+	dns.Status = dnsv1alpha1.ClusterDNSStatus{
+		ClusterIP:     clusterIP,
+		ClusterDomain: clusterDomain,
+	}
+
+	unstructObj, err := k8sutil.UnstructuredFromRuntimeObject(dns)
+	if err != nil {
+		return fmt.Errorf("failed to get unstructed obj from runtime: %v", err)
```

Comment:
Typo: "unstructed" should be "unstructured".  (In https://github.com/openshift/cluster-dns-operator/pull/76/commits/2f85300f92b76f3723b5fb7b01d3ba136aff0e34#diff-f33ebabd902db5bc0032360e4c24c3c8R97, I used the phrasing "failed to convert"—not sure what phrasing most useful in the logs.)